ethcore/types/src/errors/block_error.rs

```diff
@@ -0,0 +1,143 @@
+use std::{
```

Comment:
missing license header (and in others `*_error.rs` files), also consider removing `_error` suffix, as it's already in `error` module.